### ISSUE TITLE
```bazel run``` broken with 0.2.3

### ISSUE THREAD

**Issue by smparkes** [bazelbuild/rules_scala#52]
Haven't had time to look at this so if anyone has:
With bazel 0.2.3:
```
~/hello_world.scala git:master ❯❯❯ bazel build app
INFO: Found 1 target...
Target //:app up-to-date:
  bazel-bin/app
INFO: Elapsed time: 26.085s, Critical Path: 1.79s
~/hello_world.scala git:master ❯❯❯ bazel run app
INFO: Found 1 target...
Target //:app up-to-date:
  bazel-bin/app
INFO: Elapsed time: 0.467s, Critical Path: 0.00s

INFO: Running command line: bazel-bin/app
/private/var/tmp/_bazel_smparkes/08fe4a003be085753c237916361d8505/hello_world.scala/bazel-out/local-fastbuild/bin/app: line 3: /private/var/tmp/_bazel_smparkes/08fe4a003be085753c237916361d8505/hello_world.scala/bazel-out/local-fastbuild/bin/app.runfiles/external/local_jdk/bin/java: No such file or directory
ERROR: Non-zero return code '127' from command: Process exited with status 127.
```
```
./7e7209a62ba3a685a6bd98b8dbd655ea/scala/bazel-out/local-fastbuild/bin/app.runfiles/__main__/external/local_jdk/bin/java
```
and
```
./7e7209a62ba3a685a6bd98b8dbd655ea/scala/bazel-out/local-fastbuild/bin/app.runfiles/local_jdk/bin/java
```
exist but
```
/private/var/tmp/_bazel_smparkes/08fe4a003be085753c237916361d8505/hello_world.scala/bazel-out/local-fastbuild/bin/app.runfiles/external/local_jdk/bin/java
```
does not

**Comment by johnynek** [comment]
Yes, can fix this tomorrow I hope.

The issue was also there in 0.2.2 but it had some OK there issue so I
skipped dealing with it.

On Tuesday, May 17, 2016, Steven Parkes <notifications@github.com> wrote:

> Haven't had time to look at this so if anyone has:
> With bazel 0.2.3:
>
> ~/hello_world.scala git:master ❯❯❯ bazel build app
> INFO: Found 1 target...
> Target //:app up-to-date:
>   bazel-bin/app
> INFO: Elapsed time: 26.085s, Critical Path: 1.79s
> ~/hello_world.scala git:master ❯❯❯ bazel run app
> INFO: Found 1 target...
> Target //:app up-to-date:
>   bazel-bin/app
> INFO: Elapsed time: 0.467s, Critical Path: 0.00s
>
> INFO: Running command line: bazel-bin/app
> /private/var/tmp/_bazel_smparkes/08fe4a003be085753c237916361d8505/hello_world.scala/bazel-out/local-fastbuild/bin/app: line 3: /private/var/tmp/_bazel_smparkes/08fe4a003be085753c237916361d8505/hello_world.scala/bazel-out/local-fastbuild/bin/app.runfiles/external/local_jdk/bin/java: No such file or directory
> ERROR: Non-zero return code '127' from command: Process exited with status 127.
>
> ./7e7209a62ba3a685a6bd98b8dbd655ea/scala/bazel-out/local-fastbuild/bin/app.runfiles/__main__/external/local_jdk/bin/java
>
> and
>
> ./7e7209a62ba3a685a6bd98b8dbd655ea/scala/bazel-out/local-fastbuild/bin/app.runfiles/local_jdk/bin/java
>
> exist but
>
> /private/var/tmp/_bazel_smparkes/08fe4a003be085753c237916361d8505/hello_world.scala/bazel-out/local-fastbuild/bin/app.runfiles/external/local_jdk/bin/java
>
> does not
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bazelbuild/rules_scala/issues/52>
>


-- 
P. Oscar Boykin, Ph.D. | http://twitter.com/posco | http://pobox.com/~boykin

**Comment by johnynek** [comment]
should be fixed by #54
